cli/command/environment.go

```diff
@@ -269,21 +288,31 @@ func (e *EnvironmentCommand) unlink(c *cli.Context) error {
 	}
 
 	if sourceEnvironmentID == destEnvironmentID {
-		return fmt.Errorf("Cannot unlink an environment from itself")
+		return fmt.Errorf("Cannot link an environment to itself")
 	}
 
-	req := models.DeleteEnvironmentLinkRequest{
-		SourceEnvironmentID: sourceEnvironmentID,
-		DestEnvironmentID:   destEnvironmentID,
-	}
+	addLinkFN := func(sourceEnvID, destEnvID string) error {
+		updateEnvReq, err := generateReq(sourceEnvID, destEnvID)
+		if err != nil {
+			return err
+		}
 
-	jobID, err := e.client.DeleteLink(req)
-	if err != nil {
-		return err
+		jobID1, err := e.client.UpdateEnvironment(sourceEnvID, updateEnvReq)
```

Comment:
[nit] there's only one job in scope, so `jobID` is probably a more appropriate var name here.

Reply:
✅